.ci/build.sh

```diff
@@ -9,14 +9,16 @@ mkdir private _site public
 
 META_REVIEW_DATA="meta_review.json"
 
-EXPORTED_DATA="static/tasks.yaml static/instances.yaml static/$META_REVIEW_DATA"
+EXPORTED_DATA="static/tasks.yaml static/instances.yaml static/$META_REVIEW_DATA\
+			  static/tasks_no_issue.yaml"
 
 ISSUES_JSON="issues.json"
 
 python manage.py fetch_deployed_data --allow-failure _site $EXPORTED_DATA
 
 if [[ -n "$GCI_TOKEN" ]]; then
   python manage.py fetch_gci_task_data private
+  python manage.py gci_no_issue_tasks private _site	
```

Comment:
this should be run after the cleanse, otherwise it risks publishing private information.